net-mgmt/pfSense-pkg-arpwatch/Makefile

```diff
@@ -0,0 +1,45 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-arpwatch
+PORTVERSION=	0.1.0
+PORTREVISION=	0
+CATEGORIES=	net-mgmt
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	julego@gmail.com
+COMMENT=	Arpwatch package for pfSense
+
+LICENSE=	APACHE20
+
+RUN_DEPENDS=	${LOCALBASE}/sbin/arpwatch:net-mgmt/arpwatch
```

Comment:
Replace `${LOCALBASE}/sbin/arpwatch` by `arpwatch>=0`. Current ports syntax requests package name instead of random binary path